amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/proxy/v2/AmqpProxyClientChannel.java

```diff
@@ -0,0 +1,220 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.streamnative.pulsar.handlers.amqp.proxy.v2;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.qpid.server.bytebuffer.QpidByteBuffer;
+import org.apache.qpid.server.protocol.v0_8.AMQShortString;
+import org.apache.qpid.server.protocol.v0_8.FieldTable;
+import org.apache.qpid.server.protocol.v0_8.transport.BasicConsumeOkBody;
+import org.apache.qpid.server.protocol.v0_8.transport.BasicContentHeaderProperties;
+import org.apache.qpid.server.protocol.v0_8.transport.ChannelCloseOkBody;
+import org.apache.qpid.server.protocol.v0_8.transport.ClientChannelMethodProcessor;
+import org.apache.qpid.server.protocol.v0_8.transport.ExchangeDeclareOkBody;
+import org.apache.qpid.server.protocol.v0_8.transport.QueueBindOkBody;
+import org.apache.qpid.server.protocol.v0_8.transport.QueueDeclareOkBody;
+import org.apache.qpid.server.protocol.v0_8.transport.QueueUnbindOkBody;
+
+
+@Slf4j
+public class AmqpProxyClientChannel implements ClientChannelMethodProcessor {
+
+    private final Integer channelId;
+    private final ProxyBrokerConnection conn;
+
+    public AmqpProxyClientChannel(Integer channelId, ProxyBrokerConnection conn) {
+        this.channelId = channelId;
+        this.conn = conn;
+    }
+
+    @Override
+    public void receiveChannelOpenOk() {
+        conn.getChannelState().put(channelId, ProxyBrokerConnection.ChannelState.OPEN);
+        conn.getProxyConnection().getServerChannelMap().get(channelId).initComplete();
+    }
+
+    @Override
+    public void receiveChannelAlert(int replyCode, AMQShortString replyText, FieldTable details) {
+        // nothing to do
+    }
+
+    @Override
+    public void receiveAccessRequestOk(int ticket) {
+        // nothing to do
+    }
+
+    @Override
+    public void receiveExchangeDeclareOk() {
+        conn.getClientChannel().writeAndFlush(new ExchangeDeclareOkBody().generateFrame(channelId));
+    }
+
+    @Override
+    public void receiveExchangeDeleteOk() {
```

Comment:
The client needs to be notified of the result.

Reply:
This method will be supported later.